presentation/slideshow/5_functionality_of_modules.md

```diff
@@ -29,21 +29,25 @@ to the `FlightData` topic on Kafka
 
 * converts data into a GeoJSON
 
-* sends it to `Visualizaion` topic on Kafka
+* sends it to `Visualization` topic on Kafka
 
 -----------
 
 ## Visualization
 
 * **consumer** - reads data from `Visualization` topic
  and sends data to browser through socket
-   
-* **flight visualization** - recives data from socket
+
+* **flight visualization** - receives data from socket
```

Comment:
I think "receives data through socket" would be more suitable than "from socket"